include/openssl/core_dispatch.h

```diff
@@ -88,14 +88,17 @@ OSSL_CORE_MAKE_FUNC(int, core_set_error_mark, (const OSSL_CORE_HANDLE *prov))
 # define OSSL_FUNC_CORE_CLEAR_LAST_ERROR_MARK  9
 OSSL_CORE_MAKE_FUNC(int, core_clear_last_error_mark,
                     (const OSSL_CORE_HANDLE *prov))
-# define OSSL_FUNC_CORE_POP_ERROR_TO_MARK     10
+#define OSSL_FUNC_CORE_CLEAR_LAST_CONSTANT_TIME 10
```

Comment:
Definitely not!  These numbers are part of our public API and are immutable once released (which happened recently with 3.0).

I.e. `OSSL_FUNC_CORE_POP_ERROR_TO_MARK` must remain `10`.  `OSSL_FUNC_CORE_OBJ_ADD_SIGID` must remain `11` and `OSSL_FUNC_CORE_OBJ_CREATE` must remain `12`.

Reply:
Like @paulidale said.

I can understand that you'd like the line to be located here, aesthetically speaking.  You can!  Just give it another number, like 13.
Yes, it means that the numbers will be disordered, but that's still a viable choice.  That, or locate this line after the `OSSL_FUNC_CORE_OBJ_CREATE` line if you prefer numeric order.

Reply:
> > I'm wondering whether it would be at all possible to expose the error stack to FIPS provider code? This would enable us to use the `err_clear_last_constant_time` trick to optionally raise errors in constant time.
> 
> Not directly it isn't. It would require a new call back defined in `include/openssl/core_dispatch.h`. I feel a little hesitant about adding it -- these call backs are public and we've tried to limit their scope.
> 
> Why exactly does `err_clear_last_constant_time` need to be exported from libcrypto? It doesn't seem to be called from anywhere else. It is called via `err_clear_last_constant_time()` from the FIPS provider but that is indirect and shouldn't require the symbol to be exported.

`err_clear_last_constant_time` is used in `ossl_cipher_unpadblock`. The symbol would not be resolved without that export.